src/meta/meta_service.cpp

```diff
@@ -276,9 +276,11 @@ bool meta_service::try_lock_meta_op_status(meta_op_status op_status)
     return true;
 }
 
-void meta_service::unlock_meta_op_status()
+void meta_service::unlock_meta_op_status(bool is_manual)
 {
-    ddebug_f("UNLOCK meta op status from {}", enum_to_string(_meta_op_status.load()));
+    ddebug_f("UNLOCK meta op status from {}, is_manual {}",
```

Comment:
` ddebug_f("UNLOCK meta op status from {}, is_manual = {}",  enum_to_string(_meta_op_status.load()),
             is_manual);`

Reply:
I wrote as you say, but modified by clang-format. It is not bad in multi lines.

Reply:
I mean add a `=` after is_manual